app/components/polls/statements/CardButton.tsx

```diff
@@ -20,10 +22,33 @@ type CardButtonProps<C extends ChoiceEnum | number> = {
 export const CardButton = <C extends ChoiceEnum | number>({
   choice,
   choiceText,
+  activeChoice,
   onResponse,
   highlight,
   withTooltip = false,
 }: CardButtonProps<C>) => {
+  const [isHovering, setIsHovering] = React.useState(false);
+  const choiceEmoji = getChoiceEmoji(choice as Response["choice"]);
+
+  /**
+   * Check if the specified type of choice is active
+   * @param type
+   */
+  const isActiveChoiceType = (type: string) =>
+    [choice, activeChoice].every((c) => c === type) ||
+    (isHovering && type === choice);
```

Comment:
Try not to name variables `type` if you can avoid it. The ts compiler can handle it but it gets a little hard to read

Reply:
Alright noted. Thanks.